README.md

```diff
@@ -40,19 +58,13 @@ If your DHC address is different from the default `http://localhost:10000`, edit
 VITE_PROXY_URL=http://<dhc-host>:<port>
 ```
 
-## Package Overview
+## Debugging from VSCode
 
-There are many packages located in the [packages](./packages) directory. A few of the more important ones are:
-
-- [@deephaven/code-studio](./packages/code-studio): Main web UI used with the [deephaven-core](https://github.com/deephaven/deephaven-core/) backend. This package is the main front-end application, and depends on almost all other packages in the repository. It is often the easiest way to see the effect of your changes by opening this application. Follow the instructions in the [code-studio README.md](https://github.com/deephaven/web-client-ui/blob/main/packages/code-studio/README.md) to get it started.
-- [@deephaven/components](./packages/components): Component library used within the web UI.
-- [@deephaven/grid](./packages/grid): High-performance grid component used to display large tables of data.
-- [@deephaven/dashboard](./packages/dashboard/): Dashboards used in [@deephaven/code-studio](./packages/code-studio) for displaying and organizing panels.
-- [@deephaven/golden-layout](./packages/golden-layout): Layout framework used in [@deephaven/dashboard](./packages/dashboard/).
+We have a pre-defined launch config that lets you set breakpoints directly in VSCode for debugging browser code. The `Launch Code Studio` config will launch a new Chrome window that stores its data in your repo workspace. With this setup, you only need to install the React and Redux devtool extensions once. They will persist to future launches using the launch config.
 
-## Contributing
+If you are not using Chrome (e.g. Chromium on Linux), then you add a new configuration to VSCode and copy the launch config from [`settings.json`](./.vscode/settings.json). Then add the `runtimeExecutable` prop to point to your browser executable. VSCode unfortnuately does not merge workspace `settings.launch` with workspace `launch.json`, so if we add more launch configs you would need to copy to your `.vscode/launch.json` file to get the configs.
```

Comment:
For whatever reason this doesn't work for me (it also didn't before). I'm going to worry about it after I update my OS though (been meaning to do that for a while).

Reply:
Ya I don't have a Linux box to test on, so if you get around to figuring it out and there's more needed, then update this later